cves/CVE-2013-1896.yml

```diff
@@ -80,9 +93,12 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: |
+    No unit tests were added as a result of this change, this might be a good
+    thing to test for in integration testing since it involves sending certain
+    types of requests to the system as a whole.
+  code: false
```

Comment:
You mentioned that no unit tests were made for the fix, was there a unit test run when the feature was initially developed?

Reply:
Added a bit of clarification